### PR TITLE
Add pAI movement freedom when emagged

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request

New emagged pAIs improvements:
- Disables the range restriction
- Renamed the `Malicious Software Detected` button to `Reset Software` since it resets the software to default state
- Added a new law notification and flavor texts

## Why It's Good For The Game

More unique emag effects are good.  Plus this adds some polish to the current emag effects so the UI is smoother.

## Changelog
:cl:
add: Add pAI movement freedom when emagged.
qol: The "Malicious Software Detected" button has been appropriately renamed to "Reset Software" for when pAIs are emagged
qol: Add a new law notification popup and flavor texts when pAI is emagged
/:cl:

### CODE REVIEW

**Comment by TheVekter** [comment]
Does hitting the "reset software" button restore the tether functionality?

**Comment by timothymtorres** [comment]
Yes.  It also wipes all the syndicate configured master and laws.

File: code/modules/pai/pai.dm [hunk 1]
```diff
@@ -335,6 +335,9 @@
 	master_dna = "Untraceable Signature"
 	// Sets supplemental directive to this
 	laws.supplied[1] = "Do not interfere with the operations of the Syndicate."
+	leash.set_distance(INFINITY) // Freedom!!!
```
Comment (Mothblocks) [inline code/modules/pai/pai.dm:338]: Delete the component instead.

**Review by Mothblocks — COMMENTED** [review]
Shouldn't be wasting work

File: code/modules/pai/pai.dm (hunk 1 above)
Comment (timothymtorres) [reply code/modules/pai/pai.dm]: Applied with https://github.com/tgstation/tgstation/pull/77325/commits/33ac110cdd7662596174f575e6a93e967115eb72

File: code/modules/pai/pai.dm [hunk 2]
```diff
@@ -334,7 +334,10 @@
 	master_name = "The Syndicate"
 	master_dna = "Untraceable Signature"
 	// Sets supplemental directive to this
-	laws.supplied[1] = "Do not interfere with the operations of the Syndicate."
+	add_supplied_law(0, "Do not interfere with the operations of the Syndicate.")
+	qdel(leash) // Freedom!!!
```
Comment (Mothblocks) [inline code/modules/pai/pai.dm:338]: QDEL_NULL

Comment (Mothblocks) [reply code/modules/pai/pai.dm:338]: that reminds me i dont think i qdel_null this on destroy either :P

Comment (Mothblocks) [reply code/modules/pai/pai.dm:338]: then make sure stuff can handle it being null

Comment (timothymtorres) [reply code/modules/pai/pai.dm]: Applied with https://github.com/tgstation/tgstation/pull/77325/commits/54a276db6ad271ec5157dbada5c4b1a9ec5c876f

**Pull request merged**